backend/app/donation/routes.py

```diff
@@ -1,6 +1,109 @@
 from flask import request, abort, jsonify
-from app.models import Donation
 from . import donation
+from .. import db
+from app.models import Donation
+import uuid
+from datetime import datetime
+
+# Get all donations of user_id
+@donation.route("/donations/<user_uuid>", methods=["GET"])
```

Comment:
@celinehuang, should I remove this since #105 was closed, and this is no longer necessary?

Reply:
No @aqghawa we don't need this anymore, thanks fo catching it!